src/utils/recordTranslator.js

```diff
@@ -1,146 +0,0 @@
-import Core from './../core';
```

Comment:
It seems this was not addressed yet, because `RecordTranslator` is still mentioned 5x in https://github.com/handsontable/handsontable/blob/fbd71ef5d788778a7c03debe8838d8a8c073b596/handsontable.d.ts

Reply:
Eh, sorry 🤦‍♂. Thanks for that. Now it should be changed: 8d02550dd5b796045cf006299e650a298fd5f10f.